lib/evse_security/evse_security.cpp

```diff
@@ -574,14 +574,21 @@ std::string EvseSecurity::generate_certificate_signing_request(LeafCertificateTy
     std::string csr;
     CertificateSigningRequestInfo info;
 
+    // TODO(ioan): get this from the parameter when the interface will support it
+    bool use_tpm = false;
+
```

Comment:
Is this still relevant?

Reply:
Yes, still relevant will become a parameter when we will modify the interface.